docker-entrypoint.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+function stop {
+	kill $(ps -C ts3server -o pid= | awk '{ print $1; }')
+	exit
+}
+
+trap stop INT
+trap stop TERM
+
+test -d /data/files || mkdir -p /data/files && chown teamspeak:teamspeak /data/files
+ln -sf /data/files $TS_DIRECTORY/files
+
+test -d /data/logs || mkdir -p /data/logs && chown teamspeak:teamspeak /data/logs
+ln -sf /data/logs $TS_DIRECTORY/logs
+
+ln -sf /data/query_ip_whitelist.txt $TS_DIRECTORY/query_ip_whitelist.txt
```

Comment:
No the idea was that they will be always linked, because for example `ts3server.ini`:
- if you have the file in the mounted `/data`folder it will be used.
- if the server will be started with `createinifile=1`and the file does not exists in `/data` the file will be created through the symlink and the file will be created in the `/data` folder.

So its more dynamic, same for all other files.